### PR TITLE
feat(query): added "App Service Managed Identity Disabled" for Terraform

### PR DESCRIPTION
**Pull request by rafaela-soares**
**Proposed Changes**
- Added "App Service Managed Identity Disabled" for Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by felipe-avelar — DISMISSED** [review]
LGTM

**Review by nunoocx — DISMISSED** [review]

**Review by JoaoDuarteGomes — CHANGES_REQUESTED** [review]
The category should be "Resource Management" and the severity should be "High".

File: assets/queries/terraform/azure/app_service_managed_identity_disabled/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "b61cce4b-0cc4-472b-8096-15617a6d769b",
+  "queryName": "App Service Managed Identity Disabled",
+  "severity": "MEDIUM",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/app_service_managed_identity_disabled/metadata.json:4]:
```suggestion
  "severity": "HIGH",
```

File: assets/queries/terraform/azure/app_service_managed_identity_disabled/metadata.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "category": "Insecure Configurations",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/app_service_managed_identity_disabled/metadata.json:5]:
```suggestion
  "category": "Resource Management",
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Review by JoaoDuarteGomes — DISMISSED** [review]
LGTM.

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Pull request merged**